lib/fastlane/plugin/dynatrace/actions/dynatrace_action.rb

```diff
@@ -56,10 +56,21 @@ def self.run(params)
         end
 
         # iOS/tvOS workflow
-        dtxDssClientPath = Helper::DynatraceHelper.get_dss_client(params)
+        unless OS.mac?
+          UI.user_error! "A macOS machine is required to process iOS or tvOS symbols."
+        end
 
-        if !OS.mac?
-          UI.user_error! "A macOS machine is required to process iOS symbols."
+        dtxDssClientPath = Helper::DynatraceHelper.get_dss_client(params)
+        dtxDssClientDir = File.dirname(dtxDssClientPath)
+
+        Helper::DynatraceSymlinkHelper.delete_existing_lldb_symlinks(dtxDssClientDir)
+        customLLDBFrameworkPath = params[:customLLDBFrameworkPath]
+        if Helper::DynatraceSymlinkHelper.path_exists?(customLLDBFrameworkPath)
```

Comment:
Please add a log for the case that a custom path is set, but doesn't exit. Right now it just skips over the block without any hint.